src/web/routes/application/phone-number/validate.js

```diff
@@ -0,0 +1,28 @@
+const { check } = require('express-validator/check')
+const { parsePhoneNumberFromString } = require('libphonenumber-js')
+
+// TODO DW HTBHF-1564 Update validation messages (awaiting them from UX)
+const validatePhoneNumber = (phoneNumber, { req }) => {
+  if (!phoneNumber) {
+    throw new Error(req.t('validation:missingPhoneNumber'))
+  }
+
+  const parsedPhoneNumber = parsePhoneNumberFromString(phoneNumber, 'GB')
+
+  if (parsedPhoneNumber) {
+    if (parsedPhoneNumber.isValid() && phoneNumber.match(/^\+?[\d()\- ]+$/)) {
```

Comment:
I would probably combine these two separate `if` statements into a single function, something like:
```
const validFormat = /^\+?[\d()\- ]+$/
const isParsedPhoneNumberValid = (number) => ! isNil(number) && number.isValid() && number.match(validFormat)
```

Reply:
Updated to avoid nested if.